docs/engines/flink/flink-dml.md

```diff
@@ -189,4 +189,48 @@ Hint Options
-| other table parameters                           | (none)        | String   | No                                                                                                                               | All parameters of an Amoro table can be dynamically modified through SQL Hints, but they only take effect for this specific task. For the specific parameter list, please refer to the [Table Configuration](../configurations/). For permissions-related configurations on the catalog, they can also be configured in Hint using parameters such as [properties.auth.XXX in catalog DDL](../flink-ddl/#flink-sql)                                                                                                                                                                                                                                                                                                                                                                                                             |
\ No newline at end of file
+| other table parameters                           | (none)        | String   | No                                                                                                                               | All parameters of an Amoro table can be dynamically modified through SQL Hints, but they only take effect for this specific task. For the specific parameter list, please refer to the [Table Configuration](../configurations/). For permissions-related configurations on the catalog, they can also be configured in Hint using parameters such as [properties.auth.XXX in catalog DDL](../flink-ddl/#flink-sql)                                                                                                                                                                                                                                                                                                                                                                                                             |
+
+### Mixed Format Table LookUp Join With SQL
```

Comment:
```suggestion
## Lookup join with SQL
```

I suggest using H2 title here to maintain consistency with the same level of hierarchy for both Quering and Writing.
And since the documents here are all oriented towards the mixed format tables, we may not need to emphasize it again in the title.